src/main/java/ui/components/pickers/MilestonePickerState.java

```diff
@@ -12,14 +13,13 @@
  * which determine the result of the user's input
  */
 public class MilestonePickerState {
-    private List<PickerMilestone> currentMilestonesList;
-
-    public MilestonePickerState(List<PickerMilestone> milestones) {
-        currentMilestonesList = cloneList(milestones);
-    }
+    public static final int BEST_MATCHING_LIMIT = 5;
```

Comment:
Oops, I was supposed to make the height of the dialog relative to this variable value. If that's the case then yes.